src/main/js/account/account-service.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Account service to abstract the logic to interact with the bank account API.
+ *
+ * @author Xiaobing Hou
+ * @date 06/03/2022
+ */
+
+import axios from "axios";
+
+const PATH = "/api/v1/bank-accounts";
+
+class AccountService {
+    /**
+     * To get all bank accounts .
+     *
+     * @author Xiaobing Hou
+     * @date 06/03/2022
+     */
+    findAllBankAccount = () => {
+        const token = sessionStorage.getItem('trackrToken')
```

Comment:
We are not going to touch the token in JS. My previous PR will ensure the browse automatically send the JWT with every Http request. Please remove all code handling the token.

Reply:
OK. have done